docs/resources/offer.md

```diff
@@ -16,7 +16,7 @@ A resource that represent a Juju Offer.
 resource "juju_offer" "myoffer" {
```

Comment:
The `docs/resources/index` should also be updated to remove the `:hidden:` bit, otherwise this page is empty.